### ISSUE TITLE
Reduce blank spaces

### ISSUE THREAD

**Issue by oneseventhree** [Makin-Things/platinum-weather-card#68]
**Is your feature request related to a problem? Please describe.**
Is there a way to reduce the amount of blank space(red box) and make it light the card on the right:
![image](https://user-images.githubusercontent.com/15043282/200727402-0ac6f360-86c7-4d25-82f3-324d30911315.png)

I'm happy to edit the CS or whatever needs to be done :)

**Comment by Makin-Things** [comment]
Reducing that space would result in the icon overlapping the summary. Basically the text field is able to expand to the full width of the card and a lot of other icons take more space (ie. the rain icons have the cloud in the same place but with rain below it).
I am considering adding a flag to change to a more compressed format. That would basically change the text field to be left justified (currently centered), but place it to the right of the icon aligned to the bottom.
So this
![image](https://user-images.githubusercontent.com/91187456/201044609-675f522f-9e42-47b3-bff2-eb2f61c9e2d2.png)
would end up looking more like this
![image](https://user-images.githubusercontent.com/91187456/201044811-92c56fd6-0d7c-4a7d-a173-861128f343da.png)
Thoughts?

**Comment by oneseventhree** [comment]
> Reducing that space would result in the icon overlapping the summary. Basically the text field is able to expand to the full width of the card and a lot of other icons take more space (ie. the rain icons have the cloud in the same place but with rain below it).
> I am considering adding a flag to change to a more compressed format. That would basically change the text field to be left justified (currently centered), but place it to the right of the icon aligned to the bottom.
> So this
> ![image](https://user-images.githubusercontent.com/91187456/201044609-675f522f-9e42-47b3-bff2-eb2f61c9e2d2.png)
> would end up looking more like this
> ![image](https://user-images.githubusercontent.com/91187456/201044811-92c56fd6-0d7c-4a7d-a173-861128f343da.png)
> Thoughts?

Ah yes, that makes a lot of sense with the icons. It’s sunny here so my icons look too far up. 

I think the change you suggested is really good! And looks a lot better +1 for me!

**Comment by oneseventhree** [comment]
Either what you said above or how about in the centre:
![07C357A6-C2D8-499E-AEB7-BC6EAF2AD339](https://user-images.githubusercontent.com/15043282/201049722-05f868dd-c616-4e84-9c13-1abbabc9ea5f.jpeg)

**Comment by Makin-Things** [comment]
I think moving it up their would then just result in someone else asking to reduce the space even more.

**Comment by oneseventhree** [comment]
> I think moving it up there would then just result in someone else asking to reduce the space even more.

That’s true.. if only the icons weren’t so height heavy. Either way is better than what it is now :)

**Comment by Makin-Things** [comment]
I reduced them as much as possible when I created the revised icons. Unfortunately a lot of the time not much is happening under the sun/moon. I tried to keep the location of those elements withing the icon contestant, which results in quite a few having a fair bit of vacant space toward the bottom.

**Comment by oneseventhree** [comment]
> I reduced them as much as possible when I created the revised icons. Unfortunately a lot of the time not much is happening under the sun/moon. I tried to keep the location of those elements withing the icon contestant, which results in quite a few having a fair bit of vacant space toward the bottom.

isnthe sun


> I reduced them as much as possible when I created the revised icons. Unfortunately a lot of the time not much is happening under the sun/moon. I tried to keep the location of those elements withing the icon contestant, which results in quite a few having a fair bit of vacant space toward the bottom.

Ah yes. Is the sun and moon centred in the square too? 

I used the normal weather card with a theme called “waves” that turned weather icons into animations and I never noticed the centring issues with that

Either way your card is so great and thank you!

**Comment by Makin-Things** [comment]
They were based off a set of icons that had the sun towards the top left and the moon towards the top right.

**Comment by Albuca** [comment]
> Reducing that space would result in the icon overlapping the summary. Basically the text field is able to expand to the full width of the card and a lot of other icons take more space (ie. the rain icons have the cloud in the same place but with rain below it). I am considering adding a flag to change to a more compressed format. That would basically change the text field to be left justified (currently centered), but place it to the right of the icon aligned to the bottom. Thoughts?

Would love this, Right now with clear weather it looks like there is a missive space

**Comment by Soukyuu** [comment]
How about allowing to choose icon size as an alternative? They're svg, so should be losslessly scalable and I personally don't need huge icons anyway - space is precious.

In any case, thanks for creating this card, this is so far the only customizable enough AND with good documentation/UI editor
